frontend/src/app/modules/projects/create-project-modal/create-project-modal.component.ts

```diff
@@ -39,7 +39,11 @@ export class CreateProjectModalComponent extends BaseComponent implements OnInit
 
     public createForm() {
         this.projectForm = this.fb.group({
-            projectName: ['', [Validators.required, Validators.minLength(3), Validators.maxLength(50)]],
+            projectName: ['', [
+                Validators.required,
+                Validators.minLength(3),
+                Validators.maxLength(50),
+                Validators.pattern(/^(?![\u0410-\u044F\u0400-\u04FF]).*$/)]],
```

Comment:
if you add the validator this way, there will not be any error message. Add error message to app-input component and new validation function.